### ISSUE TITLE
[Feature]: Show QR code of own public keys on iPad

### ISSUE THREAD

**Issue by veryangru** [monal-im/Monal#1259]
### Precondition

- [X] I understood that this template is only for feature requests and not for bug reports

### Cross-checked related issues?

- [X] I have cross-checked this overview https://github.com/monal-im/Monal/issues/322 as well as filtered for related labels https://github.com/monal-im/Monal/labels

### Describe your feature

On iPad with Monal 6.4.4, I can't figure out how to show my own public keys as a QR code, so it can be scanned by others in order to verify the keys.

A workaround is to create a MUC, go to its settings and select "Encryption Keys". It then shows me all MUC members, including me, with an icon to display the QR code.

Am I missing something obvious or does this really not exist somewhere else in the app?

**Comment by lissine0** [comment]
Go to settings -> account -> Encryption Keys (OMEMO)
There's a QR code icon in the top right. That's what you want.

Out of curiosity, where did you expect to find your own keys?

**Comment by veryangru** [comment]
Thank you for the quick response.

> Go to settings -> account -> Encryption Keys (OMEMO)
> There's a QR code icon in the top right. That's what you want.

I've actually checked there too and couldn't find any such QR code icon in the top right. Attached is what I can see there. I've redacted my fingerprints / identifiers and JID. There is that icon that looks like a tablet, which I guess should show the type of my own device, pressing it however does nothing.

![IMG_0041](https://github.com/user-attachments/assets/d4aa32e2-aa78-44d0-a7d1-9d24383f8b5f)

> Out of curiosity, where did you expect to find your own keys?

Where you have described it would be a good place and is probably the first place I've looked through, after which I've looked through the options within a one on one chat.
